pages/src/pages/brokenWebsite/broken.tsx

```diff
@@ -0,0 +1,17 @@
+export default function BrokenWebsite() {
+  return (
+    <div className="flex flex-col items-center justify-center min-h-screen text-center bg-secondary">
+      <h1 className="text-9xl font-bold mb-4">Oops</h1>
```

Comment:
```suggestion
      <h1 className="text-9xl mb-4">Oops</h1> 
```
// css classes of children should be inherited from parent